backend/station_manager/shared/src/main/java/com/StationManager/shared/domain/train_station/TicketOffice.java

```diff
@@ -22,7 +22,7 @@ public class TicketOffice {
     private Direction direction;
     private Iterable<ServeRecord> transactions;
 
-    public final Queue<Event> events = new LinkedList<>();
+    public final Queue<com.StationManager.shared.domain.events.Event> events = new LinkedList<>();
```

Comment:
maybe import it first? and then just reference the `Event`?

Reply:
I do that in last comment

Reply:
commit*

Reply:
sorry, missed that